docs/meta/comparisons.md

```diff
@@ -50,13 +50,17 @@ than PyPDF2. See [history of pypdf](history.md).
   [QPDF]: https://github.com/qpdf/qpdf
 
 
-## pdfminer
+## pdfminer.six and pdfplumber
 
 [`pdfminer.six`](https://pypi.org/project/pdfminer.six/) is capable of
 extracting the [font size](https://stackoverflow.com/a/69962459/562769)
 / font weight (bold-ness). It has no capabilities for writing PDF files.
 
-## pdfrw / pdfminer / pdfplumber
+[`pdfplumber`](https://pypi.org/project/pdfplumber/) is a library focused mainly on extracting data from the file. Since `pdfplumber` is built on top of `pdfminer.six`, there are **no capabilities of exporting or modifying a PDF file**. It was also mentioned in [#440 (discussions)](https://github.com/jsvine/pdfplumber/discussions/440#discussioncomment-803880). However, `pdfplumber` is capable to convert a PDF file into an image, draw lines and rectangles on the image, and save it as an image file.
```

Comment:
```suggestion
[`pdfplumber`](https://pypi.org/project/pdfplumber/) is a library focused on extracting data from PDF documents. Since `pdfplumber` is built on top of `pdfminer.six`, there are **no capabilities of exporting or modifying a PDF file** (see [#440 (discussions)](https://github.com/jsvine/pdfplumber/discussions/440#discussioncomment-803880)). However, `pdfplumber` is capable of converting a PDF file into an image, [draw lines and rectangles on the image](https://github.com/jsvine/pdfplumber#drawing-methods), and save it as an image file.
```